integration/http/template

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Apologies that it's confusingly named - this dir houses resources that get deployed for integration testing.

I would consider this part of a config for the app. We have a place for such configs in `assets` BUT I am actually keen to remove as many files like this as possible.

Seems like we only use this in one test, can we just define it as a string there?